internal/framework/types/duration.go

```diff
@@ -1,18 +1,5 @@
-/*
```

Comment:
I don't think we should be touching the Go files, as this header is standard across all Flux repos. I suggest undoing all Go file changes and have some GH Action that errors out if the the Go files don't contain it. Note that the CC year will differ, and some files may contain a different license such as:   https://github.com/fluxcd/pkg/blob/main/tar/tar.go